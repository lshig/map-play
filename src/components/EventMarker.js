import React from 'react';

export default function EventMarker() {
  return <div className="marker" />;
}
