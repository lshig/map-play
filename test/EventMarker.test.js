import { shallow } from 'enzyme';
import React from 'react';
import toJson from 'enzyme-to-json';
import EventMarker from '../src/components/EventMarker';

describe('EventMarker', () => {
  describe('basic', () => {
    const wrapper = shallow(<EventMarker />);

    test('renders', () => {
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
