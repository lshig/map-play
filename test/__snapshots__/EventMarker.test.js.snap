// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventMarker basic renders 1`] = `
<div
  className="marker"
/>
`;
